packages/common/src/hardforks.ts

```diff
@@ -167,7 +167,7 @@ export const hardforksDict: HardforksDict = {
    * URL        : https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/osaka.md
    * Status     : Final
    */
-  osaka: {
-    eips: [2935, 6800],
+  verkle: {
+    eips: [6800],
```

Comment:
Shouldn't we have `2935, 4762` too?

Reply:
Yes on 4762. 2935 is in Prague and the Verkle fork is currently being built on top of Prague.